compiler/src/dotty/tools/dotc/transform/Pickler.scala

```diff
@@ -86,7 +86,17 @@ class Pickler extends Phase {
         pickled
       }(using ExecutionContext.global)
       def force(): Array[Byte] = Await.result(pickledF, Duration.Inf)
-      if ctx.settings.YtestPickler.value then force()
```

Comment:
Can we make this a Config option? That way it is easier to find where things should be turned on.

Reply:
Sure.